### DISCUSSION TITLE
v4.15.2 Manual -- Missing ManualContents.php

### DISCUSSION THREAD

**Discussion by bcinteractive** [timschofield/webERP#332] [Problems / Bugs]
Any idea why there is no ManualContents.php - I was trying to access manual but can't.
I tried to use the master's copy of the ManualContents.php but it's not working right so it's not compatible.

**Comment by dalers** [comment]
> Any idea why there is no ManualContents.php

Hi @bcinteractive, I don't understand. ManualContents.php is still in the master branch as can be seen from the repo code listing, is it possible it has been accidentally deleted locally?

If you have cloned the master repo, I expect "git status" will show ManualContents.php was deleted locally and if you "git pull" it should be recreated.

Looking at the repo now (2024-12-28 12:17PM MST):

![image](https://github.com/user-attachments/assets/2d11ac78-8142-4c83-8350-ae3df174d13a)

**Reply by bcinteractive** [reply]
Hi @dalers, I was referring to the stable version 4.15.2.  But don't mind this too much -- my work around was to use the PDF version of the manual.
